CONTRIBUTING.md

```diff
@@ -73,18 +93,21 @@ open http://localhost:8000/
 
 ### Publishing
 
-To publish the packages to npm, you'll need to be added as an owner for the packages you're publishing. Use the #design-system Slack channel for more information.
+To publish the packages to npm, you'll need to be added as an owner for the
+packages you're publishing. Use the #design-system Slack channel for more
+information.
 
 These Lerna commands can be helpful when publishing:
 
-- Run `npm run changed` to see which packages have changed since the last release.
+- Run `npm run changed` to see which packages have changed since the last
+release.
 - Run `npm run diff` to see a diff of all packages since the last release.
 - Run `npm run publish` to publish the updated packages with Lerna.
 
-
 ### Static Docs Site
 
-Markdown and source code for the [static docs site](https://pricelinelabs.github.io/design-system/) are located in `docs/`.
+Markdown and source code for the [static docs site][static-docs-site] are
```

Comment:
NB: when the link name is the same as the shortcut, it will automatically create a link – if you take this suggestion, it'll also need updated below

> The implicit link name shortcut allows you to omit the name of the link, in which case the link text itself is used as the name.

```suggestion
 Markdown and source code for the [docs site][] are
```